tests/config.test.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
I would say until we have editing available we should should use the ini one, unless this is just for a test.

Reply:
I left the json version for testing